00new_packages.autobuild

```diff
@@ -1,4 +1,7 @@
 # IMPORTANT: new packages must be added at the top of this file
 # They must NEVER be added to other files
 
-
+cmake_package 'gui/rock_webapp'
+ruby_package 'tools/rest_api' do |pkg|
+    Autoproj.env_add_path 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
```

Comment:
+1 for "_PATH" 

But the ROCK_PREFIX_PATH would also make the overriding for testing harder. In case the webapp to test is in a different root folder (don't know if this case occurs often at all)